cfps/CFP-002-template.md

```diff
@@ -0,0 +1,84 @@
+# CFP-002: Template
+
+**SIG: SIG-NAME**
+
+**Begin Design Discussion:** YYYY-MM-DD
+
+**End Design Discussion:** YYYY-MM-DD
+
+**Cilium Release:** X.XX
+
+**Authors:** alice <alice@example.com>, bob jones <bob@example.com>
+
+## Summary
+
+_Provide a high-level summary. Keep it short._
+
+## **Motivation**
```

Comment:
No need for bold formatting, here and below. GitHub format them the same way (`font-weight: 600`) anyway.